src/user/bot.js

```diff
@@ -84,16 +84,19 @@ class UserBot {
       , prompted  = false
 
     _.bot.on('*', (msg) => {
-      const id    = msg.from.id
+      const id  = msg.from.id
           , name  = {first: msg.from.first_name, last: msg.from.last_name}
           , uname = msg.from.username
-          , txt   = msg.text
-          , spr   = txt.split(' ')
-          , cmd   = spr[0]
-          , arg   = spr.slice(1)
-
-      console.log(msg)
-
+      if (typeof(msg.txt) == 'undefined') {
```

Comment:
We can't just take the user location input. We need a `command` as the trigger. As an example, when we ask the user for their gripe, we give `/keluhan` command then ask the the gripe.

Actually, the newest version of this bot is on the [bb24-bot-telegram](https://github.com/bb24/bb24-bot-telegram) repository. But no worries, I will makes sure that your contribution listed on the newest repository as well.